recipes-demos/ti-apps-launcher/ti-apps-launcher.bb

```diff
@@ -98,6 +98,12 @@ do_install:append() {
         install -d ${D}${systemd_system_unitdir}
         install -m 0755 ${WORKDIR}/ti-demo.service ${D}${systemd_system_unitdir}/ti-demo.service
     fi
+
+    if [ "${SERVICE_SUFFIX}" == "-eglfs" ]; then
+        install -d ${D}/etc/udev/rules.d/
+        install -m 0644 ${WORKDIR}/dev-dri-card1.rules ${D}/etc/udev/rules.d/
+    fi
+
 }
 
 FILES:${PN} += " \
```

Comment:
@sskartheekadivi Did you get a chance to build test this commit??

As I believe `FILES:${PN}` needs an update to include `${sysconfdir}`